BitwardenShared/UI/Tools/Send/Send/SendList/SendListProcessor.swift

```diff
@@ -214,13 +212,11 @@ final class SendListProcessor: StateProcessor<SendListState, SendListAction, Sen
                 type: state.type
             )
             for try await sends in result {
-                return sends
+                state.searchResults = sends
             }
         } catch {
             services.errorReporter.log(error: error)
```

Comment:
❓  It also looks like we don't have a test covering the error case; would it be worth adding one?